server/services/qBittorrent/util/torrentPropertiesUtil.ts

```diff
@@ -62,16 +62,22 @@ export const getTorrentStatusFromState = (state: QBittorrentTorrentState): Torre
       statuses.push('downloading');
       break;
     case 'metaDL':
+      statuses.push('downloading');
+      break;
     case 'downloading':
     case 'forcedDL':
       statuses.push('active');
```

Comment:
`metaDL` previously goes to here. Your change only removes `active` from the status. Is that intended? `metaDL` should be an active status.